parm/config/gfs/config.resources

```diff
@@ -163,8 +163,14 @@ case ${step} in
     export npe_wavepostpnt=200
     export nth_wavepostpnt=1
     export npe_node_wavepostpnt=$(( npe_node_max / nth_wavepostpnt ))
-    export NTASKS=${npe_wavepostpnt}
     export is_exclusive=True
+    # This MPMD job is very I/O heavy and does not scale well. Limit jobs/node to 40.
+    # For now this affects only Hercules, but will also affect Gaea.
+    if [[ ${npe_node_wavepostpnt} -gt 40 ]]; then
```

Comment:
Wouldn't this also effect WCOSS2? While this job is slow and does not scale well, I do believe it scales past 40 PEs on WCOSS2. 

This should  also be added for wavepostbndpnt and wavepostbndpntbll

Reply:
Right, I forgot about WCOSS2.  I'll add a check for Hercules instead and add the same for the other `pnt` jobs.  And since Gaea is built very similarly to WCOSS2, it may not have the same issue.  (I wonder if it is a Lustre limitation 🤔)

Reply:
So if i re-read comments correctly, this just does 40 pets per node, it's not limiting the total number of tasks correct? (sorry might need more coffee...).  If that's the case, this might also help improve things for WCOSS2 as well.  It's been a while since I did a scaling/performance study on this job.   This job does cause problems on WCOSS2 as well.

Reply:
OK, noted.  I'll change it back and run a test on WCOSS2 both ways to get that data point.

Reply:
On WCOSS2, the gfswavepostpnt job ran in 1.5 hours when executed across 2 nodes (probably 100 MPMD jobs each) vs 32 minutes when executed with 40 MPMD jobs per node.

Reply:
..... wow.... !!!!